android/src/main/java/com/google/android/cameraview/Camera1.java

```diff
@@ -86,7 +86,7 @@
 
     private String mVideoPath;
 
-    private boolean mIsRecording;
+    private final AtomicBoolean mIsRecording = new AtomicBoolean(false);
```

Comment:
How this works?

Reply:
This change was so this flag is also consistent with the photo capture one which was already atomic. Basically, this is to prevent the variable from getting changed concurrently and end up setting up a record or capture twice.